lib/caching-browserify.js

```diff
@@ -95,7 +95,8 @@ module.exports = CoreObject.extend({
       packageCache: {},
       fullPaths: this.fullPaths,
       basedir: this._inputStaging,
-      debug: this.enableSourcemap
+      debug: this.enableSourcemap,
+      browserField: !process.env.EMBER_CLI_FASTBOOT
```

Comment:
Ah, interesting. Like I said, haven't been following development too closely of late. According to https://ember-fastboot.com/docs/user-guide#packaging-your-app there's still an `app-fastboot.js` asset generated at build time. I'm not sure I follow what ends up in that bundle, though. Is it just the `fastbootDependencies` defined in your app's `package.json`?

(or are you saying that doc is out of date, and that file actually no longer gets created?)